src/js/dataEntryPanel.js

```diff
@@ -164,4 +161,20 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
         floe.chartAuthoring.percentage.render(that.locate("totalPercentage"), percentage, that.options.strings.totalPercentage);
     };
 
+    floe.chartAuthoring.dataEntryPanel.resetPanel = function (that) {
```

Comment:
The logic of this function is quite similar with what you have in floe.chartAuthoring.demo.addExampleData() @ chartAuthoring-demo.js, while the different piece is the input data. I think you probably can create a reusable invoker for both occasions. The invoker accepts the data as the argument and pass in ```{that}``` silently. Even further, if the passed in data is an empty object, it means to reset the panel.